sdk/frontdoor/arm-frontdoor/src/models/parameters.ts

```diff
@@ -20,85 +20,133 @@ export const acceptLanguage: msRest.OperationParameter = {
     }
   }
 };
-export const apiVersion: msRest.OperationQueryParameter = {
+export const aggregationInterval: msRest.OperationQueryParameter = {
+  parameterPath: "aggregationInterval",
+  mapper: {
+    required: true,
+    serializedName: "aggregationInterval",
+    type: {
+      name: "String"
+    }
+  }
+};
+export const apiVersion0: msRest.OperationQueryParameter = {
   parameterPath: "apiVersion",
   mapper: {
     required: true,
     isConstant: true,
     serializedName: "api-version",
-    defaultValue: '2018-08-01',
+    defaultValue: '2019-11-01',
     type: {
       name: "String"
     }
   }
 };
-export const backendPoolName: msRest.OperationURLParameter = {
-  parameterPath: "backendPoolName",
+export const apiVersion1: msRest.OperationQueryParameter = {
+  parameterPath: "apiVersion",
   mapper: {
     required: true,
-    serializedName: "backendPoolName",
-    constraints: {
-      MaxLength: 90,
-      MinLength: 1,
-      Pattern: /^[a-zA-Z0-9]+(-*[a-zA-Z0-9])*$/
-    },
+    isConstant: true,
+    serializedName: "api-version",
+    defaultValue: '2020-01-01',
     type: {
       name: "String"
     }
   }
 };
-export const frontDoorName: msRest.OperationURLParameter = {
-  parameterPath: "frontDoorName",
+export const apiVersion2: msRest.OperationQueryParameter = {
```

Comment:
any reason we have apiVersion, apiVersion1 and apiVersion2 ?

Reply:
I guess that's the code generator behavior of handling different apiversion.
![image](https://user-images.githubusercontent.com/55688292/79721519-10f0f480-8315-11ea-8e54-fe94694c4dac.png)
And the input include files from different version folder
![image](https://user-images.githubusercontent.com/55688292/79721661-501f4580-8315-11ea-856b-2226fe8cbd45.png)